examples/README.md

```diff
@@ -246,49 +246,49 @@ Intel® Neural Compressor validated examples with multiple compression technique
     <td>SSD ResNet50 V1</td>
     <td>Object Detection</td>
     <td>Post-Training Static Quantization</td>
-    <td><a href="https://github.com/intel/neural-compressor/tree/old_api_examples/examples/tensorflow/object_detection/tensorflow_models/quantization/ptq">pb</a> / <a href="https://github.com/intel/neural-compressor/tree/old_api_examples/examples/tensorflow/object_detection/tensorflow_models/quantization/ptq">ckpt</a></td>
+    <td><a href="https://github.com/intel/neural-compressor/tree/master/examples/tensorflow/object_detection/tensorflow_models/ssd_resnet50_v1/quantization/ptq">pb</a> / <a href="https://github.com/intel/neural-compressor/tree/master/examples/tensorflow/object_detection/tensorflow_models/ssd_resnet50_v1/quantization/ptq">ckpt</a></td>
```

Comment:
just need to provide the relative path. 
./examples/tensorflow/object_detection/tensorflow_models/ssd_resnet50_v1/quantization/ptq